CODE_OF_CONDUCT.md

```diff
@@ -55,7 +55,7 @@ further defined and clarified by project maintainers.
 ## Enforcement
 
 Instances of abuse, harassment, or otherwise unacceptable behavior may be
-reported by contacting the project team at [Ola](mailto:ogasidlo@mozilla.com) / [@zoepage](https://github.com/zoepage) or [Dennis](mailto:DSchubert@mozilla.com) / [@denschub](https://github.com/denschub). All
+reported by contacting [Ksenia](mailto:kberezina@mozilla.com) or [Dennis](mailto:DSchubert@mozilla.com) / [@denschub](https://github.com/denschub). All
```

Comment:
Maybe we can use my github handle here too ( @ksy36 )

Reply:
✅